### ISSUE TITLE
Preview: Not all settings should trigger refresh

### ISSUE THREAD

**Issue by Spreeuw** [wpovernight/woocommerce-pdf-invoices-packing-slips#307]
Currently all settings trigger a refresh for the preview. We should create a list of excluded settings and don't refresh when the setting is in that list.

**Comment by alexmigf** [comment]
### Ignore settings list

#### General
- download_display
- paper_size
- test_mode
- currency_font
- font_subsetting

#### Document
- enabled
- archive_pdf
- auto_generate_for_statuses
- attach_to_email_ids
- disable_for_statuses
- reset_number_yearly
- my_account_buttons
- invoice_number_column
- disable_free
- use_latest_settings

**Comment by alexmigf** [comment]
@Spreeuw let me know if the settings above are correct.

**Comment by Spreeuw** [comment]
Sure, but of course we can put a filter on the list and always modify if we find that there's a setting that should not trigger the update but does

**Comment by alexmigf** [comment]
@Spreeuw can you review this?

**Comment by Spreeuw** [comment]
@alexmigf 'Attach to' still triggers the refresh.
I also noticed another issue while doing this, which is that the selected order isn't kept. Here's my scenario:

1. My last order has status 'Completed', and I have selected the 'Completed' for the document, so the first time the page loads, I get the error "Document not available for order #1221, try selecting a different order."
2. I select a different order, and the document is rendered correctly (all good so far)
3. I select one of the emails in the "Attach to:" settings, which triggers a refresh (that's a bug)
4. The refresh itself is loaded with the last order (#1221) again, instead of the one I selected after seeing the error the first time, so I get the error again.
